crates/node/src/protocol.rs

```diff
@@ -901,11 +901,12 @@ where
 
                     checked!(balance - fees).map_or_else(
                         |_| {
-                            Err(Error::FeeError(
+                            Err(Error::FeeError(format!(
                                 "Masp fee payment unshielded an insufficient \
-                                 amount"
-                                    .to_string(),
-                            ))
+                                 amount: Unshielded {balance} {}, required \
```

Comment:
Can we say something like `Balance after unshield` instead of `Unshielded`? We are not technically guaranteed that the initial balance was 0